person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Your **`initialize`** method is missing the instance variable for **`@id`**. This **id** can be a randomly generated number between 1 and 1000. Kindly add it.